docs/Behaviour - Server Side.md

```diff
@@ -6,27 +6,31 @@ The initial state of Active Constraints of a Sender MUST be an empty `constraint
-
-When a Sender is in this state, it MUST NOT allow [IS-05][IS-05] activations.
+Once a Sender accepts proposed Active Constraints, this Sender, the Flow and the Connection API `/transportfile` resource (if used) associated with this Sender and the Source associated with this Flow MUST satisfy the Active Constraints when the Sender is active.
 
 ### Request Methods
 
 - `GET` request returns the last successfully applied Active Constraints.
 - `PUT` request MUST be validated and if the Sender supports Parameter Constraints used in the proposed Active Constraints and is capable to adhere at least one of the Constraint Sets, then the associated Inputs may be reconfigured, which may eventually cause the update of the corresponding Flows, Sources, `/transportfile` and Sender. If successful, the operation MUST return the accepted Active Constraints, otherwise an error MUST be returned and nothing changed.
 - `DELETE` request MUST clear Active Constraints. It turns the Sender to the initial state which excludes IS-11 out of the process.
 
-## Receivers
+## State of Sender
+
+A Sender managed with IS-11 has five states:
+- `Unconstrained` when Active Constraints of this Sender is an empty `constraint_sets` array.
+- `Constrained` when this Sender, the Flow and the `/transportfile` (if used) associated with this Sender and the Source associated with this Flow satisfy the Active Constraints.
+- `Active Constraints Violation` when this Sender, the Flow or the `/transportfile` (if used) associated with this Sender or the Source associated with this Flow does not satisfy the Active Constraints. When the Sender is in this state, it MUST NOT allow [IS-05][IS-05] activations.
+- `No Signal` when there is no signal from Inputs associated with this Sender.
+- `Awaiting Signal` when Active Constraints was just PUT and the signal from Inputs associated with this Sender is not stable yet. This is a transitional state until one of the previous ones can be established.
 
-If an active Receiver becomes unable to handle Flow, it is REQUIRED to take the following actions:
+## Preventing breaking restrictions
+
+At any time if State of the active Sender becomes `Active Constraints Violation`, the Sender MUST NOT transmit the Flow over the network. The Sender is REQUIRED to take the following actions:
 - MUST set the [IS-05][IS-05] `/active`'s `master_enable` property to `false`.
 - MUST update the [IS-04][IS-04] `subscription`'s `active` property to `false` (regardless of unicast or multicast).
 
+If Receiver activation is requested with a `/transportfile` breaking Receiver Capabilities, the Receiver is REQUIRED to refuse such request.
```

Comment:
and fail PATCH with error 400 ... (awaiting consensus from NMOS Slack channel)

Reply:
```suggestion
If `/staged` resource modification is requested for a Receiver via PATCH with a `/transportfile` violating Receiver Capabilities, the Receiver is REQUIRED to refuse such request with `400 Bad Request`.
```

Reply:
need to cover /bulk/receivers endpoint also which uses a POST and a number of receivers  ...

Reply:
```suggestion
If staged parameters update is requested for a Receiver with a `/transportfile` violating Receiver Capabilities, the Receiver is REQUIRED to refuse such request with `400 Bad Request`. If the Receiver is active, it also MUST stop receiving the Flow and REQUIRED to take the following actions:
- MUST set the [IS-05][IS-05] `/active`'s `master_enable` property to `false`.
- MUST update the [IS-04][IS-04] `subscription`'s `active` property to `false` (regardless of unicast or multicast).
```